llvm/lib/CodeGen/PreISelIntrinsicLowering.cpp

```diff
@@ -263,6 +263,19 @@ bool PreISelIntrinsicLowering::expandMemIntrinsicUses(Function &F) const {
 
       break;
     }
+    case Intrinsic::memset_inline: {
+      // Only expand llvm.memset.inline with non-constant length in this
```

Comment:
We probably should have some more intelligent heuristic / control for this, but I guess this is fine for now